tokio-console/src/config.rs

```diff
@@ -238,3 +313,67 @@ impl FromStr for RetainFor {
         }
     }
 }
+
+// === impl ColorToggles ===
+
+impl ColorToggles {
+    pub fn color_durations(&self) -> bool {
+        self.color_durations.map(std::ops::Not::not).unwrap_or(true)
+    }
+
+    pub fn color_terminated(&self) -> bool {
+        self.color_durations.map(std::ops::Not::not).unwrap_or(true)
+    }
+}
+
+// === impl ColorToggles ===
+
+impl ConfigFile {
+    fn from_config() -> Option<Self> {
```

Comment:
can we change this to return a `Result<Option<Self>, ...>`? I think that if parsing the either config file fails, we should print the error to the user. of course, if there just weren't any config files present, we should still return an `Option`.